templates/pulp-apache.conf.erb

```diff
@@ -10,7 +10,7 @@ Alias /pub /var/www/html/pub
   Require all granted
 </Location>
 
-Include /etc/pulp/vhosts80/*.conf
+Include <%= scope['apache::confd_dir'] %>pulp-vhosts80/*.conf
```

Comment:
I'm not sure whether `apache::confd_dir` does or does not contain a trailing slash, so I'd prefer to be explicit here and add one before `pulp-vhost80`.

Reply:
https://github.com/puppetlabs/puppetlabs-apache/blob/master/manifests/params.pp#L72 says it doesn't.